pkgs/applications/networking/p2p/flood-for-transmission/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib
+, stdenvNoCC
+, fetchzip
+, jq
+, commonPaths ? [ ]
+}:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "flood-for-transmission";
+  version = "2023-04-18T18-03-53";
+
+  src = fetchzip {
```

Comment:
I'm new to nixpkgs maintenance, but I see more commonly fetchFromGithub:

```nix
  src = fetchFromGitHub {
    owner = "johnam10";
    repo = pname;
    rev = version;
    sha256 = "sha256-alsHOTF8EEF7iUNHvLC21V3VRVoYQSs78g2r7YGTDeQ=";
  };
```

Reply:
I'm using `fetchzip` to download the compiled release. My understanding is that `fetchfromGitHub` fetches the source tarball.

Reply:
Ah so that is a significant difference then! Building from sources is preferred whenever possible. Imagine a problem discovered with nodejs produced frontends that makes a security issue in some packages. If you apply the patch to nodejs, it will rebuild all frontend packages and fix the problem for all.

Note that when this is merged in nixpkgs, hydra will build from source and offer users the built binary from cache.nixos.org

Reply:
Thanks for the suggestion. I was unfamiliar with the Node packaging infrastructure in Nix, but I managed to update the derivation to use `buildNpmPackage` to build the package from source.